apps/frontend/src/app/periods/[id]/columns.tsx

```diff
@@ -232,41 +234,54 @@ export const columns: (
     cell: ({ row }) => {
       const [open, setOpen] = useState(false);
       return (
-        <Popover open={open} onOpenChange={setOpen}>
-          <PopoverTrigger asChild>
+        <div className='flex gap-2 items-center justify-between'>
+          <Popover open={open} onOpenChange={setOpen}>
+            <PopoverTrigger asChild>
+              <Button
+                variant='ghost'
+                className='m-0 p-0'
+                onClick={() => {
+                  setOpen(true);
+                }}
+              >
+                <StatusBadge status={row.original.status as ApplicationStatus} />
+              </Button>
+            </PopoverTrigger>
+            <PopoverContent>
+              <Command>
+                <CommandInput placeholder='Keresés...' />
+                <CommandList>
+                  <CommandEmpty>Nincs ilyen státusz</CommandEmpty>
+                  <CommandGroup>
+                    {Object.entries(ApplicationStatus).map(([key, status]) => (
+                      <CommandItem
+                        key={key}
+                        value={status}
+                        onSelect={(value) => {
+                          onStatusChange(row.original, value as ApplicationStatus);
+                          setOpen(false);
+                        }}
+                      >
+                        <StatusBadge status={key as ApplicationStatus} />
+                      </CommandItem>
+                    ))}
+                  </CommandGroup>
+                </CommandList>
+              </Command>
+            </PopoverContent>
+          </Popover>
+          {quickMode && (row.original.status as ApplicationStatus) !== ('FINISHED' as ApplicationStatus) && (
```

Comment:
`ApplicationStatus.FINISHED` would be nicer 🤓

Reply:
en nem ertem ezt az enumot (fml), az nem mukodott, ugy mindig teljesul az egyenlotlenseg

Reply:
![image](https://github.com/user-attachments/assets/fa172038-e9fb-4027-b427-cf6962aa57df)
jaaa mert magyarul vannak az ertekek, mig backenden gondolom ugyanaz a nev meg az ertek. ez igy eleg ronda, de csak ovatosan irjatok at, ki tudja hanyszor hasznaltatok mar igy

Reply:
haat, akkor szerintem majd az osztas utan irjuk at😅 ,
de jaja tenyleg jobb lenne mappelni csak a frontenden, eleg sokat szivtam ezzel a megoldassal